sdk/identity/azure-identity/tests/test_cli_credential.py

```diff
@@ -0,0 +1,61 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import json
+
+from azure.identity import AzureCliCredential
+from azure.core.exceptions import ClientAuthenticationError
+
+from subprocess import CalledProcessError
+try:
+    from unittest.mock import Mock, patch
+except ImportError:
+    from mock import Mock, patch
+
+
+def test_cli_credential():
+    access_token = '***'
+    expires_on = '9999-1-1 00:00:00.1' 
+    mock_token = json.dumps({"accessToken" : access_token, "expiresOn" : expires_on})
+
+    with patch('azure.identity._credentials.cli_credential.check_output', return_value=mock_token):
+        cred = AzureCliCredential()
+        token = cred.get_token()
+        assert token.token == access_token
+
+def test_cli_installation():
+    cmd = ['az', 'account', 'get-access-token']
+    with patch('azure.identity._credentials.cli_credential.check_output',
```

Comment:
Patching `check_output` here but the credential now calls `run`. Does this test still pass?

Reply:
It's a mistake. Fixed now.